torchio/data/sampler/uniform.py

```diff
@@ -15,3 +17,18 @@ def __init__(self, patch_size: TypePatchSize):
 
     def get_probability_map(self, sample: Subject) -> torch.Tensor:
         return torch.ones(1, *sample.spatial_shape)
+
+    def __call__(self, sample: Subject) -> Generator[Subject, None, None]:
+
+        sample.check_consistent_spatial_shape()
+
+        if np.any(self.patch_size > sample.spatial_shape):
+            message = (
+                f'Patch size {tuple(self.patch_size)} cannot be'
+                f' larger than image size {tuple(sample.spatial_shape)}'
+            )
+            raise RuntimeError(message)
+
+        valid_range = sample.spatial_shape - self.patch_size
+        corners = np.random.randint(valid_range + 1)
```

Comment:
Hi, 
you should avoid numpy random generation as it lead to bug witch torch dataloader
use torch.rand instead